etc/kayobe/inspector.yml

```diff
@@ -62,7 +76,7 @@
 
 # Ironic inspector default network interface name on which to check for an LLDP
 # switch port description to use as the node's name.
-#inspector_lldp_switch_port_interface_default:
+inspector_lldp_switch_port_interface_default: p4p2
```

Comment:
If you need to specify multiple interfaces you can add extra rules. I can share an example if it's helpful.

Reply:
Yeah, thanks, that'd be helpful. There's an additional inspector rule I was trying to get right I'll ask about later, on retrieving the installed BIOS version at inspection too. I disabled it here as I couldn't get it working correctly.